Octo-Bot/packet-generator-bot/bot/utils/RandNetParamsGen.py

```diff
@@ -0,0 +1,69 @@
+import random
+
```

Comment:
Can you a little documentation for this function?
Including expected inputs/arguments.

Reply:
Added!